packages/client/src/generation/TSClient/Output.ts

```diff
@@ -10,15 +10,14 @@ import { wrapComment } from './helpers'
 
 export class ModelOutputField implements Generatable {
   constructor(
-    protected readonly dmmf: DMMFClass,
+    protected readonly dmmf: DMMFHelper,
     protected readonly field: DMMF.Field,
     protected readonly useNamespace = false,
   ) {}
   public toTS(): string {
     const { field, useNamespace } = this
     // ENUMTODO
-    let fieldType =
-      typeof field.type === 'string' ? GraphQLScalarToJSTypeTable[field.type] || field.type : field.type[0].name
+    let fieldType = GraphQLScalarToJSTypeTable[field.type] || field.type
```

Comment:
`field.type[0].name` can never happen, I forgot to remove this legacy code last time.